jupyter/trustyai/ubi8-python-3.8/test/test_notebook.ipynb

```diff
@@ -22,7 +22,7 @@
     "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect version\")\n",
     "\n",
     "    def test_trustyai_version(self):\n",
-    "        expected_major_minor = '0.5' # Set the expected version (x.y) \\n\n",
+    "        expected_major_minor = '0.6' # Set the expected version (x.y) \\n\n",
```

Comment:
Good catch Jan, thanks for fixing this! :slightly_smiling_face: 
/LGTM

At this point, I feel that we need to start thinking about to remove the old deprecated notebooks as well as their tests since they add a lot of unused maintenance from our side. 

I think the reason that the trustyAI test image failing on matplotlib & pandas pcg version is due to discrepancy among the notebooks. As the TrustyAI notebook is been derived from minimal->standard data science ->TrustyAI so the test follows the same chain

Reply:
Bellow are the compatible version of packages on the latest trustyAI==0.6.0
   ```
    "matplotlib~=3.6.3",
    "pandas~=1.5.3",
  ```
Ref: https://github.com/trustyai-explainability/trustyai-explainability-python/blob/0.6.0/pyproject.toml#L24-L31 

I propose until these packages aligned, to remove the datascience notebook evaluation inside trustyAI notebook.